src/FedexRest/Services/Pickup/Entity/Contact.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace FedexRest\Services\Pickup\Entity;
+
+class Contact
```

Comment:
This looks very similar to the `FedexRest\Entity\Person` class. Would it be possible to use that, or maybe extend it?

Reply:
I'll check it out.

Reply:
Now its extended from Entity/Person